client/src/components/RescheduleModal/RescheduleModal.jsx

```diff
@@ -84,23 +121,46 @@ export default class RescheduleModal extends React.PureComponent {
         );
     };
 
-    _renderContent = () => {
+    _onDateSelection = (value) => {
```

Comment:
perhaps we should add a JSDoc here that explains that value is a date (at 00:00 i assume)

```
/**
* value - a date with 00:00 time
**/
```

something like that...